app/src/main/java/com/example/evently/data/AccountDB.java

```diff
@@ -0,0 +1,172 @@
+package com.example.evently.data;
+
+import com.example.evently.data.model.Account;
+import com.google.firebase.firestore.CollectionReference;
+import com.google.firebase.firestore.DocumentReference;
+import com.google.firebase.firestore.DocumentSnapshot;
+import com.google.firebase.firestore.FirebaseFirestore;
+
+import java.util.function.Consumer;
+
+/**
+ * The account database for fetching, storing, deleting, and updating accounts in the database
+ * @author alexander-b
+ */
+public class AccountDB {
+
+    // Reference to the accounts collection
+    private final CollectionReference accountsRef;
+
+    /**
+     * Sets up the collection of accounts for the AccountDB
+     */
+    public AccountDB() {
+        // Defines the reference to the accounts collection
+        FirebaseFirestore db = FirebaseFirestore.getInstance();
+        accountsRef = db.collection("accounts");
+    }
+
+    /**
+     * Stores an account in the database.
+     * @param a The account stored in the database.
+     */
+    public void storeAccount(Account a, Consumer<Void> onSuccess, Consumer<Exception> onException) {
```

Comment:
I'm guessing you have this method take in callbacks for testing purposes but there should be a version of this method that doesn't take any callbacks.